changelog.rst

```diff
@@ -0,0 +1,7 @@
+=========
+Changelog
+=========
+
+Changes in version 1.1.0
+
+- ``ExplainCollection`` now importable from top level like so: ``from pymongoexplain import ExplainCollection``
```

Comment:
Can you add a changelog entry for 1.0.0?

Reply:
Done.